NitroxPatcher/Patches/Dynamic/QuickSlots_Bind_Patch.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Reflection;
+using HarmonyLib;
+using NitroxClient.GameLogic;
+using NitroxModel.Core;
+
+namespace NitroxPatcher.Patches.Dynamic
+{
+    //TODO: The binding should only be send on a timer and/or on disconnect. But this functionality is not implemented yet.
```

Comment:
Problem is that when we only send it on disconnect it won't save if the internet just cuts of. So I thought to prevent this we could send the quickslotBinding every 5 or 10 minutes and on proper disconnect.
Because it's just used when the player logs in I don't see the priority to send it everytime it changes.

Reply:
Would be useful to have this comment on in the code instead, for extra context.